app/route_rangers_api/templates/dashboard.html

```diff
@@ -1,64 +1,95 @@
+      var iconUrl = "{% static 'images/map_pin.png' %}";
+      var routes = {{ routes | safe}};
+      // "safe" parameter needed to deal with escaped quotation marks in GeoJSON
+      initializeMap(coordinates, stations, iconUrl, routes);
+    </script>
+        <script>
+        </script>
     </div>
-  </div>
-  <div class="col-12 col-md-4 col-lg-4">
-    <div class="card">
-      <div class="card-content">
-        <p class="card-title">Total Routes</p>
-        <p class="card-text">{{TotalRoutes}}</p>
-      </div>
+    
+    <div class="headers">
+        <h2 >Heat Map</h2>
+        <p class="inline">Maps of showing variation in transit, environmental, and socio-economic variables across census tracts </p>
+        <div id="heatmap" style="width: 600px; height: 400px;"></div>
     </div>
-  </div>
-  <div class="col-12 col-md-4 col-lg-4">
-    <div class="card">
-      <div class="card-content">
-        <p class="card-title">Average Commute Time</p>
-        <p class="card-text">{{Commute}}</p>
-      </div>
+
+
+    <div class="headers">
+        <h2 >Trends Over Time</h2>
+        <p class="inline">The following show evolution of trends in ridership and other key variables over time. This is critical as we continue to understand the impact of the pandemic.  </p>
+        <div class="container">
+            <div class="row">
+              <div class="col-6">
+            <div id="my_dataviz"></div>
+            <button onclick="drawgraph('{{csv}}','group','var1')">Bus</button>
+            <button onclick="drawgraph('{{csv}}','group','var2')">Train</button>
+            </div>
+            <div class="col-6">
+            <div id="trend_dataviz"></div>
+              </div>
+              </div>
+              </div>
     </div>
-  </div>
-</div>
 
-<div class="headers">
-  <h2>Transit Map</h2>
-  <p class="inline">Map showing all transit routes in your city</p>
-  <div id="map"></div>
-  <script type="module">
-    import { initializeMap } from "{% static 'map.js' %}";
-    var coordinates = {{ coordinates }};
-    var stations = {{ stations | safe }};
-    var iconUrl = "{% static 'images/map_pin.png' %}";
-    var routes = {{ routes | safe}};
-    // "safe" parameter needed to deal with escaped quotation marks in GeoJSON
-    initializeMap(coordinates, stations, iconUrl, routes);
-  </script>
-</div>
+<script src="https://d3js.org/d3.v4.js"></script>
+<script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>
+<script src="https://d3js.org/d3-geo-projection.v2.min.js"></script>
+<script type="module">
 
-<div class="headers">
-  <h2>Heat Map</h2>
-  <p class="inline">
-    Maps of showing variation in transit, environmental, and socio-economic variables across census
-    tracts
-  </p>
-</div>
+</script>
+<script src="{% static 'graphs.js' %}"></script>
+<script src="{% static 'heatmap.js' %}"></script>
+<script src="https://leafletjs.com/examples/choropleth/us-states.js"></script>
+<script>
```

Comment:
also think this may be more correct than what i did but we should be consistent about calling JS functions at the end of the template or within it